core/server/worker/src/main/java/alluxio/worker/dora/PagedDoraWorker.java

```diff
@@ -185,7 +186,10 @@ private void register() throws IOException {
             Configuration.getConfiguration(Scope.WORKER));
         LOG.info("Worker registered with worker ID: {}", mWorkerId.get());
         String dbDir = Configuration.getString(PropertyKey.DORA_WORKER_METASTORE_ROCKSDB_DIR);
-        int ttl = Configuration.getInt(PropertyKey.DORA_WORKER_METASTORE_ROCKSDB_TTL);
+        Duration duration =
+            Configuration.getDuration(PropertyKey.DORA_WORKER_METASTORE_ROCKSDB_TTL);
```

Comment:
@huanghua78 can you help check if getDuration can process the negative value correctly?